### ISSUE TITLE
How to Enable Debug Logging for Docker?

### ISSUE THREAD

**Issue by wschwa** [rogerfar/rdt-client#436]
The documentation says:
A log file is written to your persistent path as rdtclient.log. When you run into issues please change the loglevel in your docker script to Debug.

Can you ELI5 how to do this? I tried googling around and cannot figure out if you're referring to a generic Docker option, or something specific to this container.

I tried adding logging>options>loglevel: Debug and logging>options>log-level: Debug but neither loglevel nor log-level are recognized options.

Appreciate any help!

**Comment by rogerfar** [comment]
If you set the loglevel in RDT to debug:
![image](https://github.com/rogerfar/rdt-client/assets/1142915/621b2be3-a02b-4bf5-9f46-19f421f8ab07)
Then you can simply extract the logs from docker, lets say portainer:

![image](https://github.com/rogerfar/rdt-client/assets/1142915/2c33f5d8-1955-4a4a-97c2-e8df7ad57a35)

**Comment by wschwa** [comment]
Can't believe I missed that in the settings. Thank you so much for your prompt response!
